build-tools/src/main/java/org/elasticsearch/gradle/testclusters/RunTask.java

```diff
@@ -109,6 +111,17 @@ public Boolean getUseHttps() {
         return useHttps;
     }
 
+    @Option(option = "transport-https", description = "Helper option to enable HTTPS on transport port")
```

Comment:
can you call this `transport-tls`  avoiding HTTPS here and variable names. Transport is a proprietary protocol directly on TCP (not directly related to HTTP)